packages/carequality-sdk/src/client/carequality.ts

```diff
@@ -0,0 +1,147 @@
+import BadRequestError from "@metriport/core/util/error/bad-request";
+import { MetriportError } from "@metriport/core/util/error/metriport-error";
+import axios, { AxiosInstance, AxiosResponse } from "axios";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import { STU3Bundle, stu3BundleSchema } from "../models/bundle";
+import { OrganizationList } from "../models/organization";
+
+dayjs.extend(duration);
+
+const DEFAULT_AXIOS_TIMEOUT = dayjs.duration(120, "seconds").asMilliseconds();
```

Comment:
I'd leave the constant as duration, and when we use it we convert accordingly. Otherwise everywhere that relies on it still has to assume the unit is milliseconds